### PR TITLE
docs: Docker Toolbox for Windows 10 Home

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Setting up Docker on Windows Home is not complicated, but it's not entirely obvious if you are not already familiar with Docker, VirtualBox and specifics of using them on Windows.  Since it took me a while to figure out, I think it might help save others time if we added this to the docs.

Any comments on terminology and language in general would be great, as would confirmation that it works on someone else's machine!

### CODE REVIEW

**Comment by RandellDawson** [comment]
Not sure if we should have a separate doc just for Window Home.

**Comment by ojeytonwilliams** [comment]
Perhaps not, though I was trying to avoid making the main docs too wordy (since this isn't something everyone needs to read).  How about burying the new instructions near the bottom and changing the link to take you down to them?

**Review by raisedadead — APPROVED** [review]
